app/src/main/java/sopt/org/millie/presentation/bookdetail/BookDetailScreen.kt

```diff
@@ -0,0 +1,221 @@
+                onCompletedRateClick = onCompletedRateClick,
+                onAgeGenderClick = onAgeGenderClick,
+                onImageClick = onImageClick,
+            )
+        }
+    }
+}
+
+@Composable
+private fun BookDetailContent(
+    uiState: BookDetailState,
+    onBackButtonClick: () -> Unit,
+    onCompletedRateClick: () -> Unit,
+    onAgeGenderClick: () -> Unit,
+    onImageClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier
+            .fillMaxSize()
+            .background(color = MillieTheme.colors.bookDetailBackground)
+            .verticalScroll(state = rememberScrollState()),
+    ) {
+        Spacer(modifier = Modifier.height(22.dp))
+
+        BookDetailTopbar(
+            bookDetailTopbarBackgroundColor = MillieTheme.colors.bookDetailBackground,
+            onBackButtonClick = onBackButtonClick,
+        )
+
+        AsyncImage(
+            model = uiState.bookDetailUiModel.bookCoverImageUrl,
+            contentDescription = null,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 84.dp, vertical = 46.dp)
+                .clip(shape = RoundedCornerShape(8.dp))
+                .customShadow(
+                    shape = RoundedCornerShape(8.dp),
+                    color = MillieTheme.colors.black.copy(
+                        alpha = 0.1f,
+                    ),
+                    offsetX = 4.dp,
+                    offsetY = 8.dp,
+                )
+                .customShadow(
+                    shape = RoundedCornerShape(8.dp),
+                    color = MillieTheme.colors.black.copy(
+                        alpha = 0.2f,
+                    ),
+                    offsetX = 12.dp,
+                    offsetY = 8.dp,
+                ),
+            contentScale = ContentScale.Crop,
+        )
+
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .clip(RoundedCornerShape(topStart = 28.dp, topEnd = 28.dp))
+                .customShadow(
+                    shape = RoundedCornerShape(topStart = 28.dp, topEnd = 28.dp),
+                    color = MillieTheme.colors.black.copy(
+                        alpha = 0.1f,
+                    ),
+                    offsetY = (-5).dp,
+                )
+                .background(color = MillieTheme.colors.background)
+                .padding(top = 40.dp),
+        ) {
+            BookInfoSection(
+                bookTitle = uiState.bookDetailUiModel.bookTitle,
+                bookAuthor = uiState.bookDetailUiModel.bookAuthor,
+                bookType = uiState.bookDetailUiModel.bookType,
+                publishDate = uiState.bookDetailUiModel.publishedDate,
+                totalReviewCount = uiState.bookDetailUiModel.totalReviewCount,
+                bookRate = uiState.bookDetailUiModel.bookRate,
+                completionRate = uiState.bookDetailUiModel.completionRate,
+            )
```

Comment:
```suggestion
           with(uiState.bookDetailUiModel) {
                BookInfoSection(
                    bookTitle = bookTitle,
                    bookAuthor = bookAuthor,
                    bookType = bookType,
                    publishDate = publishedDate,
                    totalReviewCount = totalReviewCount,
                    bookRate = bookRate,
                    completionRate = completionRate,
                )
            }
```
요렇게 쓰면 가독성이 더 좋아질 것 같숭 ㅎㅎㅎㅎ

Reply:
좋네요!!
반영해놓겠습니다!